server/src/app/constants/generateJwt.js

```diff
@@ -0,0 +1,17 @@
+import jwt from 'jsonwebtoken';
+
+const generateJwt = (id, email) => {
+  const token = jwt.sign(
+    {
+      id,
+      email
+    },
+    process.env.JWT_KEY,
```

Comment:
use all the env variables from  a config/constants file.

Reply:
Done